### DISCUSSION TITLE
Some failure in MATLAB 2023a，while the same code can work on MATLAB 2016

### DISCUSSION THREAD

**Discussion by ST31Orange** [yalmip/YALMIP#1324] [General discussion]
![image](https://github.com/yalmip/YALMIP/assets/77521021/5a860027-d617-49d3-b3d9-3af2ee21325a)
![image](https://github.com/yalmip/YALMIP/assets/77521021/e0b3ad06-a042-465b-b0cf-341856e46f5d)

**Comment by johanlofberg** [comment]
My guess is that you have a "linprog" solver which isn't optimization toolbox linprog. Hence, make sure linprog really is what yalmip thinks it is (which linprog -all)

If it really is optim toolbox linprog, then your installation simply doesn't work, which you can test calling linprog examples without yalmip in the middle

**Comment by pierre-haessig** [comment]
For the reference, this is the same problem as in https://github.com/yalmip/YALMIP/discussions/1318: linprog in Matlab 2023a or 2023b has a breaking API change.

[EDIT: wrong link to discussion]
